protocols/direct-fund/direct-fund.go

```diff
@@ -224,12 +225,13 @@ func (s DirectFundObjective) fundingComplete() bool {
 
 // safeToDeposit returns true if the recorded OnChainHoldings are greater than or equal to the threshold for safety.
 func (s DirectFundObjective) safeToDeposit() bool {
+	fmt.Println(s.myDepositSafetyThreshold, s.C.OnChainFunding)
 	for asset, safetyThreshold := range s.myDepositSafetyThreshold {
+
 		chainHolding, ok := s.C.OnChainFunding[asset]
 
 		if !ok {
-			// If there are no holdings on chain we use 0 for our calculations
-			chainHolding = big.NewInt(0)
+			panic("nil chainHolding for asset in myDepositSafetyThreshold")
```

Comment:
This whole block is probably unnecessary, since we will get a panic anyway if we don't use the `,ok` pattern on L231.

Reply:
While strictly not needed, the error is probably helpful for debugging :)